### PR TITLE
fix: Separate module for erc20 migrations

### PR DESCRIPTION
**Pull request by yakischik**

### CODE REVIEW

**Review by dmitry-saraykin — APPROVED** [review]

File: .run/erc20-migration-test.run.xml
Comment (mystdeim) [inline .run/erc20-migration-test.run.xml:1]: Do we need this file in repo?

File: cicd/deploy/optimism/values.yaml [hunk 1]
```diff
@@ -32,6 +32,23 @@ optimism-erc20-listener:
       xmx_megabytes: 356
       max_direct_memory_megabytes: 32
 
+optimism-erc20-migration:
+  image:
+    repository: ethereum-indexer-erc20-migration
+  environments:
+    APPLICATION_ENVIRONMENT: '{{ include "app.env" $ }}'
+    CONSUL_ROOT_PATH: '{{ include "app.env" $ }}/protocol/{{ .Values.global.blockchain }}/erc20'
```
Comment (mystdeim) [inline cicd/deploy/optimism/values.yaml:40]: Can we get rid of consul?

File: erc20/migration/src/test/java/com/rarible/protocol/erc20/migration/test/IntegrationTest.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import com.rarible.core.test.ext.MongoTest
+import org.springframework.boot.test.autoconfigure.json.AutoConfigureJson
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ActiveProfiles
+import org.testcontainers.junit.jupiter.Testcontainers
+
+@Retention
+@MongoTest
+@KafkaTest
```
Comment (mystdeim) [inline erc20/migration/src/test/java/com/rarible/protocol/erc20/migration/test/IntegrationTest.kt:13]: I think we don't need kafka here

File: .run/erc20-migration-test.run.xml
Comment (yakischik) [reply .run/erc20-migration-test.run.xml:1]: It allows to share test runners in IDEA. And it nice if you decide to change your laptop

File: cicd/deploy/optimism/values.yaml (hunk 1 above)
Comment (yakischik) [reply cicd/deploy/optimism/values.yaml:40]: For now - no

File: erc20/migration/src/test/java/com/rarible/protocol/erc20/migration/test/IntegrationTest.kt (hunk 2 above)
Comment (yakischik) [reply erc20/migration/src/test/java/com/rarible/protocol/erc20/migration/test/IntegrationTest.kt]: Removed

**Review by mystdeim — APPROVED** [review]

**Pull request merged**